src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -130,7 +130,7 @@ const LinkOutToBridge = styled(ExternalLink)`
   padding: 6px 8px;
   text-decoration: none !important;
   width: 100%;
-  z-index: ${Z_INDEX.hover};
+  z-index: ${Z_INDEX.deprecated_zero};
```

Comment:
Deleting the z-index would just undo this [pr](https://github.com/Uniswap/interface/pull/6670) and cause the pointer bug to come back

https://github.com/Uniswap/interface/assets/35351983/096767f3-109f-4311-a229-bc256922aa18

Reply:
Can you solve this by either reducing the z-index to 1 (default), or else changing the text so that it doesn't capture the pointer (and the pointer falls back to the anchor)?